eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/AbstractHTTPServer.java

```diff
@@ -298,7 +347,11 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpRequest httpRequest)
                 processEventMeshRequest(ctx, asyncContext);
             } catch (Exception ex) {
                 httpServerLogger.error("AbrstractHTTPServer.HTTPHandler.channelRead0 err", ex);
-                // todo span end with exception.
+                //todo span end with exception.
+                span.addEvent("End Exceptional");
```

Comment:
we should add an exception here, but we need to record the cause in event atttribute

Reply:
i am not sure whether i am doing it in a right way